cypress/integration/v2/blocks/icon.spec.js

```diff
@@ -247,3 +248,41 @@ function advancedTab( viewport ) {
 	iconBlock.assertFrontendStyles()
 }
 
+function blockSpecificTests() {
+	it( 'should execute block specific tests', () => {
+		// Test 1: @see https://github.com/gambitph/Stackable/issues/1206
+
+		cy.setupWP()
+		cy.newPage()
+		cy.addBlock( 'ugb/icon' ).as( 'iconBlock' )
+		const iconBlock = registerBlockSnapshots( 'iconBlock' )
+		cy.openInspector( 'ugb/icon', 'Style' )
+		cy.toggleStyle( 'Title' )
+
+		// Set the block alignment to align center in toolbar
+		cy.adjustToolbar( 'Align', () => {
+			cy
+				.get( '.components-popover__content' )
+				.contains( 'Align center' )
+				.click( { force: true } )
+		}, {
+			buttonOrder: '0',
```

Comment:
will it work if we change the parentSelector instead of adding `buttonOrder`?

Reply:
there are no unique classes in the parent div and the button itself, so that won't work

Reply:
parentSelector should be able to handle this

Reply:
i can use :nth-of-type in the parent selector. Will commit 👍